metadata.mk

```diff
@@ -2,6 +2,9 @@
 # This file contains Makefile configuration parameters and metadata for this branch.
 #################################################################################################
 
+# The name of the product.
+PRODUCT_NAME=Calico
```

Comment:
I don't think metadata.mk is the right place for this - couldn't this just be compiled into the release binary? We have no requirement or ability for a single build of the release tool to release multiple products concurrently so I don't think it needs to be an argument to the tool, and given this variable is only used in one place I'd rather it be localized.

metadata.mk is appropriate for variables that are used in multiple locations, but for values that are only used in one place it's easier to follow if it's defined where it's used.